test/package.json

```diff
@@ -8,7 +8,7 @@
 		"start": "mocha"
 	},
 	"dependencies": {
-		"addon-tools-raub": "https://github.com/node-3d/addon-tools-raub.git",
+		"addon-tools-raub": "1.0.0",
```

Comment:
should test against the current codebase, better remove this entry altogether and in test/index.js use to `require('..')`